csharp/tools/MauiModelTester/NuGet.config

```diff
@@ -6,9 +6,11 @@
   <packageSources>
     <clear />
     <add key="NuGet Official" value="https://api.nuget.org/v3/index.json" />
```

Comment:
><add key="NuGet Official"  [](http://example.com/codeflow?start=4&length=26)

I think the official nuget.org is the culprit

Reply:
That is also in the root nuget.config and doesn't cause failure so it appears to be acceptable.